README.md

```diff
@@ -62,7 +62,9 @@ expect a response code of 200
 The SSL cert can be verified
 
 #### ``` be_served_from_origin ```
-```expect(url).to be_served_from_origin origin```
+```expect(url).to be_served_from_origin(origin, custom_headers)```
```

Comment:
Thanks Bianca. Need to look at the current implementation and see what's possible. But what I had in mind was replacing `url` with a `Request` object. `Request` contains the following attributes:
- A url
- A method (defaults to get)
- A map (dictionary) of headers (defaults to empty)
- An optional body

Ideally `Request` is defined agnostic of any http library (similar to a case class in Scala 😄), unless there is a Ruby library which is very commonly used for http calls (I don't know enough Ruby to tell)